src/azure-firewall/azext_firewall/_params.py

```diff
@@ -101,7 +111,7 @@ def load_arguments(self, _):
         c.argument('subnet', validator=get_subnet_validator(), help=argparse.SUPPRESS)
         c.argument('virtual_network_name', virtual_network_name_type, help='The virtual network (VNet) name. It should contain one subnet called "AzureFirewallSubnet".')
         c.argument('public_ip_address', help='Name or ID of the public IP to use.', validator=get_public_ip_validator())
-        c.argument('private_ip_address', help='IP address used by the Firewall ILB as the next hop in User Defined Routes.')
+        c.argument('private_ip_address', deprecate_info=c.deprecate(expiration='2.3.0'), help='IP address used by the Firewall ILB as the next hop in User Defined Routes.')
 
```

Comment:
Why 2.3.0?

Reply:
or can we choose 3.0.0? This argument already lost functionality. Actually we can directly delete it.